tests/services/test_spark.py

```diff
@@ -1,14 +1,20 @@
 # go three folders up
 import logging
-from ...app.services.spark_oceandb import *
 
+from ...app.services.spark_oceandb import FileLoader, SparkSession
 
 
-logging.info("Starting Spark Session!")
-try:
-    spark = FileLoader(SparkSession())
-    # load excel:Returns dataframe and its schema
-    _,schema = spark.load_excel("./data/SaleData.xlsx")
-    logging.info('Code Executed Successfully')
-except Exception as e:
-    logging.warning(e)
\ No newline at end of file
+class TestLoadData:
+    def load_excel(file_path: str = './data/SaleData.xlsx'):
+        """
+            # Test file loading functionality
+            # Default dest file provided
+        """
+        logging.info('Starting Spark Session!')
+        try:
+            spark = FileLoader(SparkSession())
+            # load excel:Returns dataframe and its schema
+            _, schema = spark.load_excel(file_path)
+            logging.info('Code Executed Successfully')

```

Comment:
No need to try here. If you want to test raise of exception, you should provide malformed data to the loader and use appropriate context manager then.
Please have a look. https://docs.pytest.org/en/latest/how-to/assert.html#assertions-about-expected-exceptions

Reply:
I don't know about context managements in python  Ill read up on it but lets just move on my code cant  be perfect. I endeavor to make later changes as per your advice.